### PR TITLE
Speed & Performance Update, fix bugs

### PR DESCRIPTION
**Pull request by Tomato6966**
Do everything as root User cause some System have sudo install disabled

### CODE REVIEW

File: installscript.sh [hunk 1]
```diff
@@ -4,43 +4,61 @@ echo -e '\e[31m■■■■■■■■■■■■■■■■■■■■■
 echo -e '\e[31mWelcome to Insall-Script made by Shoda#1966\e[0m'
 echo -e '\e[31m■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■\e[0m'
 
-# Install sudo
-apt-get install sudo
```
Comment (K1ZANA) [inline installscript.sh:8]: I would've replaced that with 
```
apt-get install curl
```
To be honest since you may need to curl some stuff

Comment (Tomato6966) [reply installscript.sh:8]: curl is already installed in the essentials, sudo is needed to swap to the root user using sudo su

Comment (Tomato6966) [reply installscript.sh:8]: Please look at the WHOLE updated src, nut just at one line, sudo and curl are both getting installed

Comment (K1ZANA) [reply installscript.sh:8]: > curl is already installed in the essentials, sudo is needed to swap to the root user using sudo su
Really? Wasnt installed for me.
> Please look at the WHOLE updated src, nut just at one line, sudo and curl are both getting installed

That was something i didnt really see, sorry

**Pull request closed without merging**